runtime/onert/core/include/ir/train/ITrainableOperation.h

```diff
@@ -39,6 +39,7 @@ class ITrainableOperation : virtual public IOperation
   virtual std::unique_ptr<ITrainableOperation> clone() const = 0;
   virtual void accept(OperationVisitor &v) const override = 0;
   virtual void accept(TrainableOperationVisitor &v) const = 0;
+  virtual bool hasTrainableParameter() const = 0;
```

Comment:
To check my understanding, `hasTrainableParameter()` means that the layer has weights(or bias) to be updated when backpropagation is done. Is it right?

Reply:
Yes that is correct.